aecore_eqc/aec_sync_eqc.erl

```diff
@@ -64,29 +67,33 @@ start_peers_post(_S, _Args, Res) ->
 
 
 %% --- Operation: start ---
+
+-define(PEER, peer).
```

Comment:
Don't really understand what the advantage is of using a macro here, but most likely because the word "peer" is used everywhere and hence a bit confusing.

Reply:
Yes.